src/components/graph/graph.ts

```diff
@@ -111,17 +111,22 @@ export default class GraphComponent {
     this._c10 = d3.scaleOrdinal(d3.schemeCategory10)
 
     this._actionStream = actionStream
+    let isSingleClick: boolean = true
```

Comment:
This approach works well.

Move this variable to a class variable, `private _lastClickWasSingle: boolean = false`. Then just that instead of this variable